pkg/client/k8s/k8s.go

```diff
@@ -789,6 +790,14 @@ func (k *K8sClient) ListOEV1alpha1CVRRaw(opts mach_apis_meta_v1.ListOptions) (re
 	return
 }
 
+// ListOEV1alpha1CVR fetches a list of CStorVolumeReplica as per the
+// provided options
+func (k *K8sClient) ListOEV1alpha1CVR(opts mach_apis_meta_v1.ListOptions) (cvrList *v1alpha1.CStorVolumeReplicaList, err error) {
```

Comment:
Remove this change since this is no longer required.

Reply:
will do it